test/functions_test.py

```diff
@@ -307,6 +326,8 @@
 50.12345
 >>> test_params["water"]
 335.12
+>>> test_params["temperature"]
+94.2
```

Comment:
Don't you want to add another test like this here:
```
>>> check_temperature_limits(test_params)
False
```

Reply:
No. Here, we are only testing the `filter_params` function, which removes the unnecessary .0 floating point.